app/src/main/java/com/ucmobiledevelopment/freeways/RetrofitClientInstance.kt

```diff
@@ -12,7 +12,7 @@ object RetrofitClientInstance {
             // has this object been created yet?
             if (retrofit == null) {
                 // create it
-                retrofit = retrofit2.Retrofit.Builder()
+                retrofit = Retrofit.Builder()
```

Comment:
I understand this is boilerplate, but specifying retrofit2 is not necessary as it has already been imported.